virttest/vt_agent/api.py

```
@@ -0,0 +1,224 @@
+        self.sock = None
+        self._connect()
+
+    def _connect(self):
+        """Establish a connection to the logger server."""
+        try:
+            self.sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            self.sock.connect((self.host, self.port))
+        except (socket.error, ConnectionRefusedError) as e:
+            LOG.error(
+                f"Error connecting to logger server at {self.host}:{self.port}: {e}"
+            )
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)